projects/core/src/util/try-normalize-http-error.ts

```diff
@@ -1,5 +1,6 @@
 /*
  * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
```

Comment:
nitpick: lets replace all 2023 licenses in our epic branch in separate PR please